### ISSUE TITLE
Lint the Code Base - Top Priority

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa#691]
We were having a large number of issues with code conflicts due to unlinted code. [We have now added automated linting to the PR process](https://github.com/PalisadoesFoundation/talawa/pull/681), but will need to lint the entire codebase to this new standard. 

[Linting will include following the steps in the automation script shown here](https://github.com/PalisadoesFoundation/talawa/commit/cbd592a1d8eaa216f7b923bd9be5562de20c0e30). Specifically:

1. [Run Dart Analyzer on the codebase](https://github.com/dart-lang/sdk/tree/master/pkg/analyzer_cli#dartanalyzer)
1. Fix all Dart Analyzer issues. This must be completed before proceeding to the next step.
1. Run the ``flutter format`` command on the code base
    1. ``flutter format --set-exit-if-changed . ``
 
Coordinate with @DeltaHarbinger and @Sagar2366 on this issue.

We will reopen PRs when this issue is resolved. We are looking for someone to work on this quickly.

**Comment by sumitra19jha** [comment]
Please assign me this issue.

**Comment by Neha62-lit** [comment]
I already opened this issue, but my PR on hold.
Issue no #400

**Comment by palisadoes** [comment]
@Neha62-lit Would Pedantic create a differences in the output when compared to the approach we are using?

**Comment by palisadoes** [comment]
@sumitra19jha Use this issue to lint the develop branch. [Use this method to automatically close the issue when the PR is completed.](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)
